src/screens/login/components/LoginForm.tsx

```diff
@@ -0,0 +1,95 @@
+import * as React from 'react';
+
+import {
+  Text,
+  View,
+  TextInput,
+  TouchableOpacity,
+  ActivityIndicator,
+} from 'react-native';
+
+import {styles} from './styles';
+import {validateLoginData} from '../validation';
+
+interface LoginFormsProps {
+  email: string;
+  setEmail: (email: string) => void;
+  password: string;
+  setPassword: (email: string) => void;
```

Comment:
```suggestion
  onEmailChange: (email: string) => void;
  password: string;
  onPasswordChange: (email: string) => void;
```

Acho que assim a nomenclatura fica mais próxima do que o próprio React (e a nossa convenção) usam.

Reply:
Com certeza, não havia me atentado a isso, vou alterar!